src/Maker/MakeDto.php

```diff
@@ -0,0 +1,322 @@
+        FileManager $fileManager,
+        ValidatorInterface $validator = null
+    ) {
+        $this->doctrineHelper = $doctrineHelper;
+        $this->fileManager = $fileManager;
+        $this->validator = $validator;
+    }
+
+    public static function getCommandName(): string
+    {
+        return 'make:dto';
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConf)
+    {
+        $command
+            ->setDescription('Creates a new "data transfer object" (DTO) class from a Doctrine entity')
+            ->addArgument('name', InputArgument::REQUIRED, sprintf('The name of the DTO class (e.g. <fg=yellow>%sData</>)', Str::asClassName(Str::getRandomTerm())))
+            ->addArgument('bound-class', InputArgument::REQUIRED, 'The name of Entity that the DTO will be bound to')
+            ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeDto.txt'))
+        ;
+
+        $inputConf->setArgumentAsNonInteractive('bound-class');
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+        if (null === $input->getArgument('bound-class')) {
+            $argument = $command->getDefinition()->getArgument('bound-class');
+
+            $entities = $this->doctrineHelper->getEntitiesForAutocomplete();
+
+            $question = new Question($argument->getDescription());
+            $question->setValidator(function ($answer) use ($entities) {return Validator::existsOrNull($answer, $entities); });
+            $question->setAutocompleterValues($entities);
+            $question->setMaxAttempts(3);
+
+            $input->setArgument('bound-class', $io->askQuestion($question));
+        }
+    }
+
+    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator)
+    {
+        $dataClassNameDetails = $generator->createClassNameDetails(
+            $input->getArgument('name'),
+            'Dto\\',
+            'Data'
+        );
+
+        $boundClass = $input->getArgument('bound-class');
+
+        $boundClassDetails = $generator->createClassNameDetails(
+            $boundClass,
+            'Entity\\'
+        );
+
+        // Verify that class is an entity
+        if (false === $this->doctrineHelper->isClassAMappedEntity($boundClassDetails->getFullName())) {
+            throw new RuntimeCommandException('The bound class is not a valid doctrine entity.');
+        }
+
+        /**
+         * Get class metadata (used to copy annotations and generate properties).
+         *
+         * @var ClassMetaData
+         */
+        $metaData = $this->doctrineHelper->getMetadata($boundClassDetails->getFullName());
+
+        // Get list of fields
+        $fields = $metaData->fieldMappings;
+
+        // The result is passed to the template
+        $addHelpers = $io->confirm('Add helper extract/fill methods?');
+        $omitGettersSetters = $io->confirm('Omit generation of getters/setters?');
```

Comment:
I'd suggest to change this feature to `Generate getters/setters? [y/N]` if possible